docs/resources/confluent_catalog_entity_attributes.md

```diff
@@ -33,7 +77,8 @@ resource "confluent_catalog_entity_attributes" "main" {
     secret = "<Schema Registry API Secret for data.confluent_schema_registry_cluster.essentials>"
   }
   
-  entity_name = "lkc-15xq83:topic_0"
+  # example: entity_name = "lkc-15xq83:topic_0"
+  entity_name = "${data.confluent_kafka_cluster.basic.id}:${confluent_kafka_topic.purchase.topic_name}"
```

Comment:
Nit discrepancy: I see below in description you have this instead
```
${data.confluent_schema_registry_cluster.essentials.id}:${confluent_kafka_cluster.basic.id}:${confluent_kafka_topic.purchase.topic_name}.
```

Reply:
That's a great catch! The value of the `entity_name` depends on the target resource, for example, there's `fka_cluster.basic.id}:${confluent_kafka_topic.purchase.topic_name}` in the _Argument Reference_ block as well:

> - `entity_name` - (Required String) The qualified name of the entity, for example, `${data.confluent_schema_registry_cluster.essentials.id}:.:${confluent_schema.purchase.schema_identifier}`, `${data.confluent_schema_registry_cluster.essentials.id}:${confluent_kafka_cluster.basic.id}:${confluent_kafka_topic.purchase.topic_name}`.

Reply:
Oh wait, you're right, there's
```
${data.confluent_schema_registry_cluster.essentials.id}:${confluent_kafka_cluster.basic.id}:${confluent_kafka_topic.purchase.topic_name}
```
in Argument Reference block but it should have been just 
```
${data.confluent_kafka_cluster.basic.id}:${confluent_kafka_topic.purchase.topic_name}
```
instead.